src/view/WorkspaceManager.js

```diff
@@ -191,6 +201,7 @@ define(function (require, exports, module) {
      */
     Panel.prototype.hide = function () {
         Resizer.hide(this.$panel[0]);
+        exports.trigger(EVENT_WORKSPACE_PANEL_HIDDEN, this.panelID);
```

Comment:
I generally prefer named stateless events. That is usually much easier to deal with.